src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -364,6 +385,18 @@ function CalendarWidget({
                                 onChange([]);
                               }
 
+                              if (
+                                date === currentlySelectedDate ||
+                                !value.some(selectedDate =>
+                                  isSameDay(
+                                    parseISO(date),
+                                    parseISO(selectedDate),
+                                  ),
+                                )
+                              ) {
+                                setAlertTrigger();
```

Comment:
This trigger is added to support scenario 2 in the figma design. Though currently VoiceOver is skipping these particular alert messages because the collapsable date button labels are being read out instead. This seems to be a bug with VoiceOver though.

We can work around it by removing the `aria-expanded` attribute on the button but there are tradeoffs with this approach. @Jenny-yw and I decided it's probably better to address this issue at a later time after we fixed other accessibility issues on this page.